web/stores/activities.ts

```diff
@@ -1,35 +1,72 @@
 import { pb } from "#imports";
 import type { Item } from "#imports";
 import { defineStore } from "pinia";
+import { ref } from "vue";
 
-export const useActivitiesStore = defineStore("activitiesStore", {
-  state: () => ({
-    items: [] as Item[],
-    item: undefined as Item | undefined,
-    itemTypes: new Set<string>(),
-  }),
-  actions: {
-    async load(filter: string) {
-      if (filter.length === 0) {
-        filter = "path ~ 'inbox/activities/%'";
-      }
+type LoadingStatus = 'idle' | 'pending' | 'success' | 'error'
+
+const MOCK_ITEM_TYPES = ["debt", "note"];
+function getRandomItemType () {
+  return MOCK_ITEM_TYPES[Math.floor(Math.random() * MOCK_ITEM_TYPES.length)];
+}
+
+function mockData (): Item[] {
+  return Array.from({length: 10}, (_, i) => {
+    const item = {
+      id: `${i}`,
+      title: `Item ${i}`,
+      type: getRandomItemType()!,
+      content: `Content ${i}`,
+      completed: Math.random() > 0.5 ? "true" : "false",
+      frontmatter: {} as any
+    }
+    if (item.type === "debt") {
+      item.frontmatter.currency = "USD"
+    }
+
+    return item
+  });
+}
+
+export const useActivitiesStore = defineStore("activitiesStore", () => {
```

Comment:
Please see updates in the main branch, there are probably going to be some merge conflcts.